docs/1-terms/0-structure-DE.adoc

```diff
@@ -0,0 +1,289 @@
+=== A 
```

Comment:
Is this the generated file? Then we should definitely not check it in but create from scratch.

Reply:
You are absolutely right, when the creation is reliable and fully integrated into the build process. 
Was not the primary intention so far.